<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCiagnik</class>
 <widget class="QDialog" name="AddCiagnik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="14" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAnuluj">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QDateEdit" name="dateEditOs1">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Data wymiany osi2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Data wygaśnięcia Przeglądu</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Data wymiany osi1</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QDateEdit" name="dateEditPrzeglad">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEditNumerRej">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Numer Rejestracyjny</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEditOs2">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Liczba kilometów na osi 2</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="lineEditOs1">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Liczba kilimetrów na osi 1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDateEdit" name="dateEditOs2">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDateEdit" name="dateEditCzytanie">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>ID kierowcy</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSpinBox" name="spinBoxIdKierowca">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Czytanie Tachografu</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="dateEditUbezpieczenie">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Data wygaśnięcia ubezpieczenia</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Data wygaśnięcia Tachografu</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QDateEdit" name="dateTachograf">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>label_7</zorder>
  <zorder>label_6</zorder>
  <zorder>label_2</zorder>
  <zorder>spinBoxIdKierowca</zorder>
  <zorder>dateEditCzytanie</zorder>
  <zorder>dateTachograf</zorder>
  <zorder>lineEditOs2</zorder>
  <zorder>dateEditOs2</zorder>
  <zorder>lineEditOs1</zorder>
  <zorder>dateEditOs1</zorder>
  <zorder>dateEditUbezpieczenie</zorder>
  <zorder>dateEditPrzeglad</zorder>
  <zorder>lineEditNumerRej</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEditNumerRej</tabstop>
  <tabstop>dateEditPrzeglad</tabstop>
  <tabstop>dateEditUbezpieczenie</tabstop>
  <tabstop>dateEditOs1</tabstop>
  <tabstop>lineEditOs1</tabstop>
  <tabstop>dateEditOs2</tabstop>
  <tabstop>lineEditOs2</tabstop>
  <tabstop>dateTachograf</tabstop>
  <tabstop>dateEditCzytanie</tabstop>
  <tabstop>spinBoxIdKierowca</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonAnuluj</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
